Dockerfile

```diff
@@ -125,6 +126,13 @@ RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
   conda activate core3d && \
   pip install -e ./danesfield"]
 
+# Install 3d-tiles-tools for converting glb to b3dm
+RUN ["/bin/bash", "-c", "git clone https://github.com/CesiumGS/3d-tiles-validator.git"]
+
+# Install gltf-pipeline for converting gltf to glb
+RUN ["/bin/bash", "-c", "git clone https://github.com/CesiumGS/gltf-pipeline.git"]
```

Comment:
Is it necessary / desirable to install these (beyond just cloning) here also?  I see installation is covered in the `tiler-readme.md` document.  Or is installation largely only necessary in certain niche use cases.

Reply:
Good catch. Not sure why I omitted that - I'll check if this is required and add it here or update the documentation.

Reply:
Yes, npm install is needed - it downloads all dependencies.